src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+const initialToDoList = [
+	{
+		task: 'Organize Garage',
+		id: 1528817077286,
```

Comment:
if you want to create long unique id's easily there is a good library ```uuid```

You just need to ```import uuid from 'uuid'``` then call ```uuid()``` when you need it